tekton/publish-nightly.yaml

```diff
@@ -75,6 +75,14 @@ spec:
         ${inputs.params.pathToProject}/${outputs.resources.builtCredsInitImage.url}: ${inputs.params.imageRegistry}/${inputs.params.pathToProject}/build-base:latest
         ${inputs.params.pathToProject}/${outputs.resources.builtGitInitImage.url}: ${inputs.params.imageRegistry}/${inputs.params.pathToProject}/build-base:latest
         ${inputs.params.pathToProject}/${outputs.resources.builtEntrypointImage.url}: busybox # image should have shell in $PATH
+      baseBuildOverrides:
+        $(input.params.pathToProject)/$(outputs.resources.builtControllerImage.url):
+          env:
+            - name: CGO_ENABLED
+              value: 1
+          flags:
+            - name: ldflags
+              value: "-X $(input.params.pathToProject)/pkg/version.PipelineVersion=$(inputs.params.versionTag)"
```

Comment:
This version of the publish task does not have versionTag as an input param. 
You will need to move the `generate-release-version` step ahead of this and use "/tekton/home/version"

Reply:
It's actually /builder/home - see https://github.com/tektoncd/pipeline/pull/1667

Reply:
Have changed it to `/builder/home` hope your's gets merged first